tests/test_algorithm.py

```diff
@@ -1905,6 +1905,21 @@ def test_arg_types(self, name, inputs):
 
         self.assertEqual(expected, cm.exception.args[0])
 
+    def test_empty_asset_list_to_history(self):
+        algo = TradingAlgorithm(
+            script=dedent("""
+                def initialize(context):
+                    pass
+
+                def handle_data(context, data):
+                    data.history([], "price", 5, '1d')
```

Comment:
are we allowing `fields` to be an empty iterable as well? If so, maybe the test should have `[]` in place of `price`